programs/client/FuzzLoop.cpp

```diff
@@ -536,22 +538,27 @@ bool Client::buzzHouse()
                 std::uniform_int_distribution<uint32_t> next_dist(1, prob_space);
                 const uint32_t nopt = next_dist(rg.generator);
 
+                if ((correctness_oracle + settings_oracle + dump_oracle + peer_oracle) > 0
+                    && nopt < (correctness_oracle + settings_oracle + dump_oracle + peer_oracle + 1))
```

Comment:
This if statement is not understandable at all. What does each of the things mean? Can we rewrite it considering intermediate boolean variables that we can easily understand?

Reply:
I can disable statements dynamically and the probability functions needs to generate the available statements with a fair probability. I can clean this a little more.